### ISSUE TITLE
Updated alernative

### ISSUE THREAD

**Issue by ishtanzar** [schapman1974/tinymongo#61]
This project had a lot of potential but since it's not maintained anymore (e.g. broken with TInyDB 4), https://github.com/davidlatwe/montydb seems very promising

**Comment by jefmud** [comment]
@ishtanzar I agree, time to move on to MontyDB.  It is more compatible with PyMongo so you can give users a choice between MontyDB for local database and PyMongo for MongoDB (local and remote).

**Comment by schapman1974** [comment]
Current state: tinymongo is active again, CI is passing, and version 1.1.0 has been published to PyPI with modern Python packaging, backend docs, and expanded compatibility tests. MontyDB may still be a good fit for some users, but this issue is not an actionable tinymongo bug or feature request, so I am closing it.
